dashboard/app/lib/javascripts/dashboard/main.js

```diff
@@ -65,6 +66,24 @@ window.Dashboard = {
 		Marbles.history.navigate('login'+ redirectPath);
 	},
 
+	__isCertInstalled: function() {
+		var self = this, dispatchIfCertIsMissing = function(a, b) {
+			if (a + b !== 200 + 200) {
```

Comment:
we can't check if the controller is up via http, because the browser is pointed at `https://dashboard.FLYNN_CLUSTER` and it's not possible to request an http resource from an https resource.